### DISCUSSION TITLE
Show mentions within discussions

### DISCUSSION THREAD

**Discussion by brandonkelly** [community/community#4301] [Discussions]
Discussions don’t currently list other issues/discussions where the discussion is referenced, in the same way that issues do.

For example, I referenced a couple related discussions in [this comment](https://github.com/craftcms/cms/discussions/9075#discussioncomment-916116), however neither of the referenced discussions have any mention of their being referenced by craftcms/cms#9075.

This is a super valuable Issues feature that’s sorely missing.

Related: #3193

**Comment by brandonkelly** [comment]
Sorry, perhaps this is a duplicate of #2971.

**Comment by cliffordfajardo** [comment]
+1 for this feature
I really like how github issues links to other issues/discussions, which helps surfaces the activity or importance user's may be giving a topic

This discussion by @brandonkelly is a prime example of a use case. @brandonkelly mentioned https://github.com/github/feedback/discussions/2971  and it would be nice to see inside https://github.com/github/feedback/discussions/2971  " @brandonkelly  mentioned this discussion: 

**Mock up:**
<img width="545" alt="CleanShot 2022-01-08 at 02 59 48@2x" src="https://user-images.githubusercontent.com/6743796/148641643-db4220f5-66ef-4a1e-ac48-138f33abde2a.png">
